lib/modules/platform/github/index.ts

```diff
@@ -1959,27 +1958,19 @@ export async function getVulnerabilityAlerts(): Promise<VulnerabilityAlert[]> {
     logger.debug('No vulnerability alerts enabled for repo');
     return [];
   }
-  let vulnerabilityAlerts: { node: VulnerabilityAlert }[] | undefined;
-
-  // TODO #22198
-  const gheSupportsStateFilter = semver.satisfies(
-    // semver not null safe, accepts null and undefined
-
-    platformConfig.gheVersion!,
-    '>=3.5',
-  );
-  const filterByState = !platformConfig.isGhe || gheSupportsStateFilter;
-  const query = vulnerabilityAlertsQuery(filterByState);
+  let vulnerabilityAlerts: VulnerabilityAlert[] | undefined;
 
   try {
-    vulnerabilityAlerts = await githubApi.queryRepoField<{
-      node: VulnerabilityAlert;
-    }>(query, 'vulnerabilityAlerts', {
-      variables: { owner: config.repositoryOwner, name: config.repositoryName },
-      paginate: false,
-      acceptHeader: 'application/vnd.github.vixen-preview+json',
-      readOnly: true,
-    });
+    vulnerabilityAlerts = (
+      await githubApi.getJson<VulnerabilityAlert[]>(
+        `/repos/${config.repositoryOwner}/${config.repositoryName}/dependabot/alerts?state=open`,
```

Comment:
is the state filter supported by all supported ghe versions? for graphql we need ghe v3.5+

Reply:
https://github.com/renovatebot/renovate/issues/26796#issuecomment-1903757887